packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -382,4 +311,35 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
       ),
     );
   }
+
+  Widget _buildInfoAddedRow(String text, {File? imgStart}) {
+    final ThemeData themeData = Theme.of(context);
+    return Padding(
+      padding: _ROW_PADDING_TOP,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(
+            height: 50,
+            width: 50,
+            child: ClipRRect(
+              borderRadius: ROUNDED_BORDER_RADIUS,
+              child: imgStart == null
```

Comment:
I think we decided to remove the image for now as we were only able to show one at a time. 

@AshAman999 do you remember what we agreed on

Reply:
I think let's not confuse with this pr , we can merge this one , perhaps will deal with that separately in a another pr.